### DISCUSSION TITLE
When I try to sbuscribe to a notification I get error #98

### DISCUSSION THREAD

**Discussion by eternalparquet** [saleweaver/python-amazon-sp-api#369] [Q&A]
When I try yo subscribe to a notification after I succesfully created a destination id I get this error:
`{'code': 'InvalidInput', 'message': 'Request has missing or invalid parameters and cannot be parsed.', 'details': 'No destination with destinationid for applicationId appID'}`
I am using python, this is the code:

```
subscr = Notifications(marketplace=Marketplaces.IT).create_subscription(
    notification_type=NotificationType.ANY_OFFER_CHANGED,
    destination_id="destinatioid"
    )
```

**Comment by saleweaver** [comment]
It seems like there is **No destination with destinationid for applicationId appID**.
Pass a destination id that exists instead, as received from the `create_destination` call

**Reply by eternalparquet** [reply]
I am passing an id created with createDestination, so it exists.

**Reply by saleweaver** [reply]
The error message indicates something different. Feel free to add an verifiable example so I can help.

**Comment by eternalparquet** [comment]
This is the destination id created:
```
{'errors': None,
 'headers': {'Date': 'Mon, 31 Jan 2022 14:49:22 GMT', 'Content-Type': 'application/json', 'Content-Length': '220', 'Connection': 'keep-alive', 'x-amzn-RequestId': 'f58647d6-3a62-49c6-9549-41b7d471345c', 'x-amzn-RateLimit-Limit': '1.0', 'x-amz-apigw-id': 'M0OF4FLdoAMFg-A=', 'X-Amzn-Trace-Id': 'Root=1-61f7f6f2-47c0e58332843fe464d5f033'},
 'kwargs': {},
 'next_token': None,
 'pagination': None,
 'payload': {'destinationId': '79f0067b-0f53-41c6-a469-e5126253a326',
             'name': 'amazon-price-adjuster',
             'resource': {'eventBridge': None,
                          'sqs': {'arn': 'xxxxx'}}},
 'rate_limit': '1.0'}
```
Now you can try to subscribe:

```
subscr = Notifications(marketplace=Marketplaces.IT).create_subscription(
    notification_type=NotificationType.ANY_OFFER_CHANGED,
    destination_id="79f0067b-0f53-41c6-a469-e5126253a326"
    )
print(subscr)
```

If I recreate the destination id:

`dest = Notifications().create_destination(name="amazon-price-adjuster", arn="arn:aws:sqs:eu-west-1:719578215409:amazon-seller-notifications") `

I get this error:

`[{'code': 'Conflict', 'message': 'The requested resource conflict with a resource in the server.', 'details': 'A destination with arn arn:aws:sqs:eu-west-1:ACCOUNTID:amazon-seller-notifications already exists for the application amzn1.sp.solution.ID'}]`

So there is a destination id for the app but there isn't when I create a subscription.
